README.md

```diff
@@ -1,110 +1,297 @@
+        - [Start up SynAnno](#start-up-synanno)
+- [Navigating SynAnno](#navigating-synanno)
+    - [Landing Page](#landing-page)
+    - [Open Data](#open-data)
+    - [Annotate](#annotate)
+    - [Categorize](#categorize)
+    - [Export Annotations](#export-annotations)
+    - [Draw](#draw)
+    - [Export Masks](#export-masks)
 
-1. Proofreading annotated data
-    - Viewing data instances and mask
-    - Marking faulty segmentation masks
-    - Describing error in segmentation mask
-2. Correcting Segmentation masks
-    - Deleting False Positives
-    - Redrawing bad matches (under development)
 
-## Requirements
+## Key Components and Subjects 
 
-**Download and unzip SynAnno**
+### H01
 
-Download SynAnno and unzip it into a folder of your choice.
-We assume you have unzipped the folder under `/home/user/SynAnno`.
+Harvard's Lichtman laboratory and Google's Connectomics team released the [H01](https://h01-release.storage.googleapis.com/landing.html) dataset, a 1.4 petabyte view of human brain tissue via nanoscale EM. It covers a volume of ~1mm³, featuring tens of thousands of neurons, millions of neuron fragments, 183 million annotated synapses, and 100 proofread cells.
 
-**Anaconda (https://www.anaconda.com/)**
+### Synaptic Polarity Annotation
 
-Conda is an environment and package manager for python built for data science and machine learning.
-It provides an easy and stable means to set up an environment with all of SynAnno's relevant dependencies.
-To run SynAnno, even the lightweight version MiniConda is sufficient.
+Synaptic polarity refers to the directionality of information flow between two neurons at a synapse, the junction where they communicate. In this context, we classify synapses as pre-synaptic (information senders) or post-synaptic (information receivers). A pre-synaptic neuron sends neurotransmitter signals across the synaptic cleft to the post-synaptic neuron, which receives these signals and processes the information. While the segmentation masks highlight the synaptic clefts between neurons, the pre-/post-synaptic IDs are coordinates placed into the associated neurons, identifying the specific sender and receiver. Identifying these key elements is crucial for creating accurate structural and functional neural maps. SynAnno assists in proofreading, correcting, and identifying these segmentation masks and synaptic IDs.
 
-To download and install MiniConda, go to [here](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links) and select the latest version for your Operation System.
-For MacOs, this would be "Miniconda3 MacOSX 64-bit pkg" or "Miniconda3 macOS Apple M1 64-bit pkg," depending on your Mac's chip architecture. Select the "... pkg" version, not the "... bash" version.
-After downloading the package, double-click it and follow the steps described by the install manager.
+### CAVE (Connectome Annotation Versioning Engine)
```

Comment:
@jakobtroidl could you please add here a short section about CAVE. What it is and its purpose?

Feal free to directly add it via the GUI and contribute to the PR.